plugin/generator/generator.go

```diff
@@ -155,7 +179,15 @@ func (g *CaddyfileGenerator) GenerateCaddyfile() ([]byte, string) {
 		logsBuffer.Write(processLogs)
 	}
 
-	return caddyfileContent, logsBuffer.String()
+	if len(caddyfileContent) == 0 {
+		caddyfileContent = []byte("# Empty caddyfile")
```

Comment:
Why this instead of empty string?

Reply:
I think if Caddy reads an empty file it returns an EOF error?

Reply:
Makes sense.